kubernetes-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/build/ApplyMojo.java

```diff
@@ -86,12 +60,6 @@
     public static final String DEFAULT_KUBERNETES_MANIFEST = "${basedir}/target/classes/META-INF/jkube/kubernetes.yml";
     public static final String DEFAULT_OPENSHIFT_MANIFEST = "${basedir}/target/classes/META-INF/jkube/openshift.yml";
 
-    /**
-     * The domain added to the service ID when creating OpenShift routes
-     */
-    @Parameter(property = "jkube.domain")
-    protected String routeDomain;
```

Comment:
Is this parameter still supported via properties?

Reply:
I have moved this to ResourceMojo since route/ingress generation is no longer part of apply goal